plaidml/op/op.h

```diff
@@ -437,6 +443,11 @@ inline edsl::Tensor minimum(const edsl::Tensor& X, const edsl::Tensor& Y) {
   return details::op("minimum", args).as_tensor();
 }
 
+inline edsl::Tensor norm(const edsl::Tensor& I, int axis, float eps, EpsMode eps_mode) {
```

Comment:
Not critical but I think `epsilon` would be a clearer variable name than `eps`. EpsMode can probably stay as-is (it's clear from context when there's a variable named `epsilon`). Also it's probably sufficient to change it at the API level (i.e. here in op.h); in the context of the implementation it's pretty clear what `eps` is.

This matters in part because an epsilon term is only needed when this is used in the denominator, and this code covers more cases than that.